tools/travis/deploy.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -e
+
+# Build script for Travis-CI.
+
+SCRIPTDIR=$(cd $(dirname "$0") && pwd)
+ROOTDIR="$SCRIPTDIR/../.."
+HOMEDIR="$ROOTDIR/.."
+WHISKDIR="$ROOTDIR/../openwhisk"
+UTILDIR="$ROOTDIR/../incubator-openwhisk-utilities"
+
+# run scancode
+cd $UTILDIR
+scancode/scanCode.py $ROOTDIR
+
+# Install OpenWhisk
+cd $WHISKDIR/ansible
+
+ANSIBLE_CMD="ansible-playbook -i environments/local"
+
+$ANSIBLE_CMD setup.yml
+$ANSIBLE_CMD prereq.yml
+$ANSIBLE_CMD couchdb.yml
+$ANSIBLE_CMD initdb.yml
+
+cd $WHISKDIR
+
+TERM=dumb ./gradlew \
+:tools:admin:install \
+:common:scala:install \
+:core:controller:install \
+:core:invoker:install \
+:tests:install \
+distDocker
+
```

Comment:
I think we have been trying to clean this part up in places. 
I believe you can run just `./gradlew install` and it will replace lines 29-33

Reply:
+1, thanks!